### DISCUSSION TITLE
1.2.0 one of my pool contract not showing up across my accounts on other computers

### DISCUSSION THREAD

**Discussion by ZielReaver** [Chia-Network/chia-blockchain#7239] [Support]
I have seen this sense 1.1.795 beta version. I created a pool and it only shows up on two of my three computers. The third computer when trying to join the pool created a whole new contract address and launcher ID and this new one has appaired on my other two computers but those computers can create plots to new contract address but the points never get sent to the pool.

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
